ios/brave-ios/Sources/AIChat/Components/Messages/AIChatIntroMessageView.swift

```diff
@@ -64,13 +120,59 @@ struct AIChatIntroMessageView: View {
         .frame(maxWidth: .infinity, alignment: .leading)
         .fixedSize(horizontal: false, vertical: true)
 
-      Text(modelDescription)
-        .font(.footnote)
-        .foregroundStyle(Color(braveSystemName: .textTertiary))
-        .multilineTextAlignment(.leading)
-        .frame(maxWidth: .infinity, alignment: .leading)
-        .fixedSize(horizontal: false, vertical: true)
-        .padding(.bottom)
+      HStack(alignment: .firstTextBaseline) {
+        Text(modelDescription)
+          .font(.footnote)
+          .foregroundStyle(Color(braveSystemName: .textTertiary))
+          .multilineTextAlignment(.leading)
+          .fixedSize(horizontal: false, vertical: true)
+          .padding(.bottom)
+
+        Button(
+          action: {
+            shouldShowInformationPopover = true
+          },
+          label: {
+            Label {
+              Text(Strings.AIChat.leoPageContextInfoDescriptionTitle)
+            } icon: {
+              Image(braveSystemName: "leo.info.outline")
+                .foregroundStyle(Color(braveSystemName: .iconDefault))
+                .font(.footnote)
+            }
+            .labelStyle(.iconOnly)
+          }
+        )
+        .bravePopover(
+          isPresented: $shouldShowInformationPopover,
+          arrowDirection: .forcedDirection(.up)
+        ) {
+          PopoverWrapperView(
+            backgroundColor: UIColor(braveSystemName: .containerBackground)
+          ) {
+            VStack {
+              Text(popoverMessage)
+                .font(.footnote)
+                .foregroundStyle(Color(braveSystemName: .textPrimary))
+                .fixedSize(horizontal: false, vertical: true)
+                .frame(maxWidth: .infinity, alignment: .leading)
+
+              Text(
+                "[Learn More](https://support.brave.com/hc/en-us/categories/20990938292237-Brave-Leo)"
```

Comment:
Not localized

Reply:
Fixed in: https://github.com/brave/brave-core/pull/23440#:~:text=Localized%20Learn%20More%20url.